api/src/Microsoft.Azure.IIoT.OpcUa.Api.Publisher/src/Models/PublishedNodeApiModel.cs

```diff
@@ -77,5 +79,10 @@ public class PublishedNodeApiModel {
         [DataMember(Name = "queueSize", Order = 11,
             EmitDefaultValue = false)]
         public uint? QueueSize { get; set; }
+        
+        /// <summary> Data change filter </summary>
+        [DataMember(Name = "dataChangeFilter", Order = 12,
+            EmitDefaultValue = false)]
+        public DataChangeTriggerType? DataChangeFilter { get; set; }
```

Comment:
>DataChangeTriggerType [](http://example.com/codeflow?start=15&length=21)

Let's define an api model for this enum and not use the internal model.

Reply:
There was already an enum defined in `api/src/Microsoft.Azure.IIoT.Api/src/Publisher/Models`; I moved it to api/src/Microsoft.Azure.IIoT.OpcUa.Api.Publisher/src/Models to avoid circular dependencies.

Reply:
Can we call the property Data change trigger?  Data change filter is larger and includes dead band, so the name will be misleading.  Or we do the entire data change filter.

Reply:
I've change it now; however, I don't understand why that name in the `PublishedDataSetVariableApiModel` (see https://github.com/Azure/Industrial-IoT/blob/898257cf0ae44f8c4b2413154d0449f11198fe2c/api/src/Microsoft.Azure.IIoT.Api/src/Publisher/Models/PublishedDataSetVariableApiModel.cs#L75), which doesn't seem to include dead band (as those are fields below).